akka-remote/src/main/java/akka/remote/artery/AeronErrorLog.java

```diff
@@ -37,26 +37,31 @@
 public class AeronErrorLog
 {
     private final File cncFile;
+    final MappedByteBuffer cncByteBuffer;
+    final DirectBuffer cncMetaDataBuffer;
+    final int cncVersion;
+    final AtomicBuffer buffer;
+    final SimpleDateFormat dateFormat;
 
     public AeronErrorLog(File cncFile)
     {
       this.cncFile = cncFile;
+      cncByteBuffer = IoUtil.mapExistingFile(cncFile, "cnc");
+      cncMetaDataBuffer = CncFileDescriptor.createMetaDataBuffer(cncByteBuffer);
+      cncVersion = cncMetaDataBuffer.getInt(CncFileDescriptor.cncVersionOffset(0));
+      buffer = CncFileDescriptor.createErrorLogBuffer(cncByteBuffer, cncMetaDataBuffer);
+      dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSSZ");
+
+
+      if (CncFileDescriptor.CNC_VERSION != cncVersion)
+      {
```

Comment:
That reminds me - when do we want to enable the java formatter? It is ready to roll.